### PR TITLE
Removed page reloading when renaming entities

### PR DESCRIPTION
**Pull request by stpavlenko**

### CODE REVIEW

File: src/ui/units/connections/components/Page/components/UnloadConfirmation.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import React from 'react';
 import {I18n} from 'i18n';
 import {useSelector} from 'react-redux';
 import {Prompt} from 'react-router';
+import {selectIsRenameWithoutReload} from 'ui/store/selectors/entryContent';
```
Comment (artemipanchuk) [inline src/ui/units/connections/components/Page/components/UnloadConfirmation.tsx:6]: It's better not to use 'ui' alias
Use relative path please

File: src/ui/units/connections/store/constants.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import {DL} from 'ui';
+import {DL} from 'ui/constants';
```
Comment (artemipanchuk) [inline src/ui/units/connections/store/constants.ts:1]: Same

File: src/ui/units/dash/components/DashActionPanel/DashActionPanel.tsx [hunk 3]
```diff
@@ -13,6 +13,7 @@ import type {DatalensGlobalState, EntryDialogues} from 'ui';
 import {ActionPanel, DL, EntryDialogName, EntryDialogResolveStatus} from 'ui';
 import {registry} from 'ui/registry';
 import {closeDialog as closeDialogConfirm, openDialogConfirm} from 'ui/store/actions/dialog';
+import {selectIsRenameWithoutReload} from 'ui/store/selectors/entryContent';
```
Comment (artemipanchuk) [inline src/ui/units/dash/components/DashActionPanel/DashActionPanel.tsx:16]: Same

File: src/ui/units/datasets/containers/Dataset/Dataset.js [hunk 4]
```diff
@@ -15,6 +15,7 @@ import {compose} from 'recompose';
 import {createStructuredSelector} from 'reselect';
 import {DatasetActionQA, ErrorCode, ErrorContentTypes} from 'shared';
 import {openDialogErrorWithTabs} from 'ui/store/actions/dialog';
+import {selectIsRenameWithoutReload} from 'ui/store/selectors/entryContent';
```
Comment (artemipanchuk) [inline src/ui/units/datasets/containers/Dataset/Dataset.js:18]: Same

File: src/ui/units/editor/components/ButtonSave/ButtonSave.js [hunk 5]
```diff
@@ -16,6 +16,7 @@ import {
     navigateHelper,
     sdk,
 } from 'ui';
+import {selectIsRenameWithoutReload} from 'ui/store/selectors/entryContent';
```
Comment (artemipanchuk) [inline src/ui/units/editor/components/ButtonSave/ButtonSave.js:19]: Same

**Review by artemipanchuk — APPROVED** [review]

File: src/ui/units/wizard/components/Dialogs/DialogPlaceholder/DialogPlaceholder.tsx [hunk 6]
```diff
@@ -28,7 +28,8 @@ import {
     isNumberField,
     isPercentVisualization,
 } from 'shared';
-import {Utils, withHiddenUnmount} from 'ui';
+import {Utils} from 'ui';
```
Comment (mournfulCoroner) [inline src/ui/units/wizard/components/Dialogs/DialogPlaceholder/DialogPlaceholder.tsx:31]: One more issue about imports( 
Can you use `ui/utils` or `ui/utils/utils`, please?

**Review by mournfulCoroner — APPROVED** [review]

**Pull request merged**